src/components/AboutMe.jsx

```diff
@@ -40,21 +41,37 @@ class AboutMe extends Component {
 
   render() {
     return (
-      <div id="divaboutme" className="jumbotron jumbotron-fluid m-0">
+      <div id="aboutme" className="jumbotron jumbotron-fluid m-0">
         <div className="container container-fluid p-5">
           <div className="row">
-            <div className=" col-5 d-none d-lg-block align-self-center">
-              <img
-                className="border border-secondary rounded-circle"
-                src={this.state.instaProfilePic}
-                alt="profilepicture"
-              />
-            </div>
-            <div className="col-lg-7">
+            {this.state.instaProfilePic !== true &&
```

Comment:
doesn't work good at "showInstaProfilePic": false,
try this:
>>>
            {this.state.instaProfilePic &&
              <div className="col-5 d-none d-lg-block align-self-center">
<<<